app/views/patterns/search-for-something/README.md

```diff
@@ -23,8 +23,8 @@ Don't provide users with an advanced search page. Users prefer to search and _th
 
 ## Research on this pattern
 
-More research is needed on this.
+We need more research. If you have used the search pattern, get in touch to share your research findings.
```

Comment:
Should we be telling people to contribute research via the issue?

Reply:
Good point, yes. I'll create an issue to update the text to get contribution via the issue.